client/extensions/woocommerce/state/ui/shipping/zones/locations/selectors.js

```diff
@@ -543,7 +543,7 @@ export const getShippingZoneLocationsList = createSelector(
 		const locations = getShippingZoneLocations( state, zoneId, siteId );
 		return getShippingZoneLocationsListFromLocations( state, locations, maxCountries, siteId );
 	},
-	( state, zoneId, maxCountries = 999, siteId = getSelectedSiteId( state ) ) => {
```

Comment:
Note that i needed to remove this initialization for the `no-unused-vars` not to fail.